pocs/utils/pyro/__init__.py

```diff
@@ -0,0 +1,65 @@
+import netifaces
+
+
+def get_own_ip(verbose=False, logger=None):
```

Comment:
Is this code being run on the remote camera?

Reply:
It is optionally used by both the camera servers (on the remote camera controllers) and the name server (can be anywhere, but probably on the main control computer), more frequently by the latter.

I don't anticipate it being used much at all in operational systems though, where it's likely that everything will have a static IP which can be put into config files, but it is handy during testing.